src/ts/mapboxMap.ts

```diff
@@ -420,14 +451,14 @@ module powerbi.extensibility.visual {
                 }
             });
 
-            this.onUpdate(this.map, this.settings, true, this.updatedHandler);
+            this.onUpdate(this.map, this.settings, true, this.settings.api.geocoder, this.updatedHandler);
```

Comment:
can we handle the onUpdate function using just `this.settings`?  We could for example store the true/false geocoder state in settings.json, then look in `this.settings` to see if the geocoder visual should be toggled on or off.